### DISCUSSION TITLE
importing MapboxGl is not recognized.

### DISCUSSION THREAD

**Discussion by saltron85** [rnmapbox/maps#2347] [Q&A] [answered]
Hey guys, I've been following the tutorial and when I write this line:

import  MapboxGL from '@rnmapbox/maps';

i get this warning "Could not find a declaration file for module '@rnmapbox/maps'."

I searched online and I cant find any solution. I even tried installing @types it suggest and its not found.

**Comment by mfazekas** [comment]
@saltron85 which version is it?! Npm or from git?

**Reply by saltron85** [reply]
I installed it with the command 
Expo install @rnmapbox/maps

It installs 10.0.0-beta.50 
In my package.json file.

**Comment by mfazekas** [comment, marked as answer]
@saltron85 can you try installing 49?! 

I think that maybe #2337 is the cause of the issue

```
expo install @rnmapbox/maps@10.0.0-beta.49
```

**Reply by saltron85** [reply]
@mfazekas thank you so much! that took care of it!
